std/experimental/ndslice/internal.d

```diff
@@ -1,3 +1,5 @@
+/// @@@DEPRECATED_2017-04@@@
+deprecated("Please use mir-algorithm DUB package: http://github.com/libmir/mir-algorithm")
```

Comment:
https is important in spite of the fact that github will redirect http to https - at least in theory a compromised network could intercept the http request and take it elsewhere...